### PR TITLE
Upgrade planner to Angular 6 and RxJS 6

### PR DESCRIPTION
**Pull request by sanbornsen**
<strong>Note: If there are pending changes to the PR, prefix the PR title with "WIP" and add the label "DO NOT MERGE"</strong>

### Mandatory
 - [x] What does this PR do? 
     This PR upgrades planner to Angular 6 and RxJS 6

- [ ] What issue/task does this PR references?

- [ ] Are the tests Included?
___
### Optional
- [ ] Is the documentation Included?

- [ ] Are the Release Notes included?

- [ ] @mention(s) to expected reviewer(s) included

### CODE REVIEW

**Comment by sanbornsen** [comment]
[test]

File: package.json [hunk 1]
```diff
@@ -83,48 +83,48 @@
     "ng2-dragula": "^1.5.0",
     "ng2-truncate": "^1.3.11",
     "ngx-bootstrap": "^1.9.3",
-    "ngx-feature-flag": "^0.6.0",
+    "ngx-feature-flag": "1.0.0",
```
Comment (rohitkrai03) [inline package.json]: We should move this to peer dependencies. That will solve the DI issue i guess.

File: package.json [hunk 2]
```diff
@@ -83,48 +83,48 @@
...
+    "@angular/platform-browser-dynamic": "^6.1.6",
+    "@angular/platform-server": "^6.1.6",
+    "@angular/router": "^6.1.6",
+    "@ngrx/effects": "^6.1.6",
+    "@ngrx/entity": "^6.1.6",
+    "@ngrx/store": "^6.1.6",
+    "@ngrx/store-devtools": "^6.1.6",
     "ngx-base": "^2.3.0",
     "ngx-fabric8-wit": "^6.20.2",
     "ngx-login-client": "1.4.5",
```
Comment (rohitkrai03) [inline package.json]: `ngx-login-client` can be upgraded to v2.5.1

File: src/app/components_ngrx/typeahead-selector/typeahead-selector.component.ts [hunk 3]
```diff
@@ -4,8 +4,7 @@ import {
   Output, ViewChild
 } from '@angular/core';
 import { BehaviorSubject, Observable } from 'rxjs';
-import { combineLatest } from 'rxjs/observable/combineLatest';
-import { of as observableOf } from 'rxjs/observable/of';
+import { combineLatest, of as ObservableOf } from 'rxjs';
```
Comment (rohitkrai03) [inline src/app/components_ngrx/typeahead-selector/typeahead-selector.component.ts]: For consistency across codebases, I would prefer if we use named exports as is without changing the name.

**Comment by sahil143** [comment]
Closing this PR. New PR: https://github.com/fabric8-ui/fabric8-planner/pull/2794

**Pull request closed without merging**